### PR TITLE
Change Metricbeat default config to reduce disk space

### PR DESCRIPTION
**Pull request by tsg**
This implements several changes from #4112:

* reduce the frequency for the fsstats and filesystem
* filter out some filesystems by mount point
* set one shard by default
* enable best_compression by default

It also removes some of the redundant comments.

Question:
* This currently modifies the template only for metricbeat. Do we want some of these changes for other beats as well?

### CODE REVIEW

File: metricbeat/docs/modules/system.asciidoc [hunk 1]
```diff
@@ -86,45 +86,30 @@ in <<configuration-metricbeat>>. Here is an example configuration:
...
   processes: ['.*']
   process.include_top_n:
     by_cpu: 5      # include top 5 processes by CPU
     by_memory: 5   # include top 5 processes by memory
+- module: system
+  enabled: true
+  period: 1m
+  metricsets:
+    - filesystem
+    - fsstat
```
Comment (ruflin) [inline metricbeat/docs/modules/system.asciidoc]: We should really rename this to `filbeatsystem_summary` ;-)

File: metricbeat/_meta/setup.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#==================== Elasticsearch template setting ==========================
+setup.template.settings:
+  index.number_of_shards: 1
```
Comment (s1monw) [inline metricbeat/_meta/setup.yml]: this should be the damned default in ES to begin with...

**Comment by s1monw** [comment]
I am just giving a LGTM for the sake of the number of shards setting!

File: metricbeat/docs/modules/system.asciidoc (hunk 1 above)
Comment (monicasarbu) [inline metricbeat/docs/modules/system.asciidoc]: I guess you mean `filesystem_summary`. I agree with you.

Comment (ruflin) [inline metricbeat/docs/modules/system.asciidoc]: lol, yes :-)

**Pull request merged**